kb/migrating-visibility-data-from-es6.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: migrate-visibility-data-from-es6
+title: Migrate visibility data from ES6
+tags:
+  - kb-article
+date: 2022-29-08T00:00:00Z
+---
+
+We added support for [Elasticsearch v7+ (ES7)](https://www.elastic.co/downloads/past-releases/#elasticsearch) in the v1.7.0 update to the [Temporal Server](/clusters#temporal-server).
```

Comment:
After this line, I believe it is worth to add that since v1.12.0 v7 is default Elasticsearch version. And probably worth to add summary table like:
1.7.0 - v7 support added, but v6 is still default,
1.12.0 - v7 is default, but v6 is still supported,
1.18.0 - v6 support is completely removed, v8 support is added.